README.md

```diff
@@ -4,221 +4,501 @@
+data = [["tom", 10, "TLV"], ["nick", 15, "Berlin"], ["juli", 14, "NY"]]
+df = pd.DataFrame(data, columns = ["name", "age", "city"])
+df.set_index("name")
+v3c.write(backend="kv", table="mytable", dfs=df, condition="age>14")
 ```
 
-### Read
-Reads data from a backend.  
-Common read parameters:
-* iterator: bool - Return iterator of DataFrames or (if False) just one DataFrame
-* filter: string - Query filter (can't be used with query)
-* columns: []str - List of columns to pass (can't be used with query)
-* data_format: string - Data format (Not yet supported)
-* marker: string - Query marker (Not yet supported)
-* limit: int - Maximal number of rows to return (Not yet supported)
-* row_layout: bool - Weather to use row layout (vs the default column layout) (Not yet supported)
-
-
-#### backend specific parameters
-##### TSDB
-* start: string
-* end: string
-* step: string
-* aggregators: string
-* aggregationWindow: string
-* query: string - Query in SQL format
-* group_by: string - Query group by (can't be used with query)
-* multi_index: bool - Get the results as a multi index data frame where the labels are used as indexes
- in addition to the timestamp, or if `False` (default behavior) only the timestamp will function as the index.
-
-For detailed info on these parameters please visit [TSDB](https://github.com/v3io/v3io-tsdb#v3io-tsdb) docs.  
-Example:
+<a id="method-read"></a>
+### read Method
+
+Reads data from a data collection (table/stream) in a platform data container to a DataFrame according to the configured backend.
+
+- [Syntax](#method-read-syntax)
+- [Common parameters](#method-read-backend-common-params)
+- [`tsdb` backend `read` parameters](#method-read-params-tsdb)
+- [`kv` backend `read` parameters](#method-read-params-kv)
+- [`stream` backend `read` parameters](#method-read-params-stream)
+
+Reads data from a backend.
+
+<a id="method-read-syntax"></a>
+#### Syntax
+
 ```python
-df = client.read(backend='tsdb', query="select avg(cpu) as cpu, avg(diskio), avg(network)from mytable", start='now-1d', end='now', step='2h')
+read(backend=<backend>, table=<table>, attrs=<backend-specific parameters>)
 ```
 
-##### KV
-* reset_index: bool - Reset the index. When set to `false` (default), the dataframe will have the key column of the v3io kv as the index column.
-When set to `true`, the index will be reset to a range index.
-* max_in_message: int - Maximal number of rows per message
-* sharding_keys: []string (Experimental)- list of specific sharding keys to query. For range scan formatted tables only.
-* segments: []int64 (Not yet supported)
-* total_segments: int64 (Not yet supported)
-* sort_key_range_start: string (Not yet supported)
-* sort_key_range_end: string (Not yet supported)
+<a id="method-read-backend-common-params"></a>
+#### Common read Parameters
+
+All Frames backends that support the `read` method support the following common parameters, which can be set in the `attrs` method parameter:
```

Comment:
`which can be set in the `attrs` method parameter:` -> that's just not true. remove this.

Reply:
Removed; will be committed shortly to the PR branch.